### ISSUE TITLE
Make AVR Programmer Software available on Flathub

### ISSUE THREAD

**Issue by Knochi** [pololu/pololu-usb-avr-programmer-v2#4]
Hi,

I'm using my [SteamDeck](https://www.steamdeck.com/en/) as gaming device but also as my tinkering/development machine (which works surprisingly well).
Now i have a project I want to use the AVR Programmer, but since steamdeck is depending on flathub, when it comes to software, I'm not able to install the AVR Programmer software to my deck.

Could you please make it available on Flathub? I would be willing to do a nice video on how I use my SteamDeck to program an A-Star board, if you like.

Have a look at their guide on how to make apps available on flathub:
https://github.com/flathub/flathub/wiki/App-Submission

**Comment by DavidEGrayson** [comment]
Hello.  Is there any particular reason you cannot download and use the binaries we provide for Linux from the "Installing Linux software" section of the [Pololu USB AVR Programmer v2 user's guide](https://www.pololu.com/docs/0J67)?

**Comment by Knochi** [comment]
Yes the steam deck uses SteamOS, which is an read-only version of Arch Linux with Plasma Desktop and a locked files system for compatibility reasons.
The only way to install apps is flathub.
